mlrun/db/httpdb.py

```diff
@@ -3264,6 +3266,43 @@ def _wait_for_background_task_to_end(bg_name):
 
         return bg_task
 
+    def get_datastore_profile(
+        self, name: str, project: str
+    ) -> mlrun.common.schemas.DatastoreProfile:
+        project = project or config.default_project
+        path = self._path_of("projects", project, "datastore_profiles") + f"/{name}"
+
+        res = self.api_call(method="GET", path=path)
+        if res and res._content:
```

Comment:
Don't we need to check the status code? Maybe instead of checking `_content`.

Reply:
It might return empty when the object does not exist and the status code will be 200 but the content will be empty

Reply:
Still we should probably check both then? Could also be an error code with content, e.g. 500.

Reply:
self.api_call() throws an exception on error, so no need to check error code